google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -192,6 +192,11 @@ def run(self):
             traceback.print_exc(file=buf)
             buf.seek(0)
             message = buf.read()
+            try:
+                self.session.commit()
+            except Exception as e:
```

Comment:
There should not be any need for such a broad except catch here, please catch the exception(s) that session.commit() could raise. That will also fix the lint error.